File: app/LoginActivity.java
package com.yourpackage

import android.content.Intent
import android.os.Bundle
import android.widget.*
import androidx.appcompat.app.AppCompatActivity

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        val email = findViewById<EditText>(R.id.etEmail)
                val password = findViewById<EditText>(R.id.etPassword)
                val loginBtn = findViewById<Button>(R.id.btnLogin)
                val registerText = findViewById<TextView>(R.id.tvRegister)

                loginBtn.setOnClickListener {
            val emailText = email.text.toString()
            val passText = password.text.toString()

            if (emailText == "user@mail.com" && passText == "123456") {
                startActivity(Intent(this, ChatListActivity::class.java))
            } else {
                Toast.makeText(this, "Login Gagal", Toast.LENGTH_SHORT).show()
            }
        }

        registerText.setOnClickListener {
            startActivity(Intent(this, RegisterActivity::class.java))
        }
    }
}

File: app/src/main/java/com/example/uts_pm/LoginActivity.kt
package com.example.uts_pm

import android.content.Intent
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        val etEmail = findViewById<EditText>(R.id.etEmail)
        val etPassword = findViewById<EditText>(R.id.etPassword)
        val btnLogin = findViewById<Button>(R.id.btnLogin)
        val btnToRegister = findViewById<Button>(R.id.btnToRegister)

        btnLogin.setOnClickListener {
            Log.d("LoginActivity", "Tombol Login ditekan")

            val email = etEmail.text.toString()
            val password = etPassword.text.toString()

            val sharedPref = getSharedPreferences("UserData", MODE_PRIVATE)
            val savedEmail = sharedPref.getString("email", null)
            val savedPassword = sharedPref.getString("password", null)

            if (email == savedEmail && password == savedPassword) {
                startActivity(Intent(this, HomeActivity::class.java))
                finish()
            } else {
                Toast.makeText(this, "Email atau Password salah", Toast.LENGTH_SHORT).show()
            }


            btnToRegister.setOnClickListener {
                Log.d("LoginActivity", "Tombol Daftar ditekan")
                Toast.makeText(this, "Menuju halaman daftar", Toast.LENGTH_SHORT).show()

                startActivity(Intent(this, RegisterActivity::class.java))
            }
        }
    }
}